src/auto/rectangle.rs

```diff
@@ -0,0 +1,5 @@
+// This file was generated by gir (aacfc92) from gir-files (71d73f0)
+// DO NOT EDIT
+
+use ffi;
+use glib::translate::*;
```

Comment:
I can't prevent `gir` from generating it? Or I should just remove it from `git` and ignore it?